.travis.mk

```diff
@@ -369,6 +369,7 @@ test_static_build_cmake_osx: base_deps_osx
 ###########
 
 deps_freebsd:
+	echo y | sudo pkg update -f
```

Comment:
1. Please, consider Sergey's comment: https://github.com/tarantool/tarantool/issues/5749#issuecomment-768401196
2. Don't forget to link the issue: #5749.